webui/react/src/pages/ExperimentDetails/ExperimentVisualization/LearningCurve.tsx

```diff
@@ -149,15 +149,15 @@ const LearningCurve: React.FC<Props> = ({
 
   const handlePointClick = useCallback(
     (e: MouseEvent, point: UPlotPoint) => {
-      const trialId = point ? trialIds[point.seriesIdx - 1] : undefined;
+      const trialId = point ? trialIds[point.seriesIdx] : undefined;
       if (trialId) handleTrialClick(e, trialId);
     },
     [handleTrialClick, trialIds],
   );
 
   const handlePointFocus = useCallback(
     (point?: UPlotPoint) => {
-      const trialId = point ? trialIds[point.seriesIdx - 1] : undefined;
+      const trialId = point ? trialIds[point.seriesIdx] : undefined;
       if (trialId) handleTrialFocus(trialId);
     },
     [handleTrialFocus, trialIds],
```

Comment:
not blocker: idk how it should work, but focus or hover doesnt highlight series. Do you know how it should work?

Reply:
<img width="1523" alt="Screenshot 2023-10-16 at 5 31 22 PM" src="https://github.com/determined-ai/determined/assets/40620519/b84272e9-123c-4ed5-a7e1-cf059d05005f">
When hovering on a point, the row for the respected trial (first if there are multiple) would be highlighted.

Reply:
ah i see. row highlight. not series highlight. thanks